app/src/main/java/org/mozilla/fenix/tabstray/viewholders/NormalBrowserPageViewHolder.kt

```diff
@@ -79,15 +78,13 @@ class NormalBrowserPageViewHolder(
         layoutManager: RecyclerView.LayoutManager
     ) {
         val concatAdapter = adapter as ConcatAdapter
-        val browserAdapter = concatAdapter.browserAdapter
         val inactiveTabAdapter = concatAdapter.inactiveTabsAdapter
         val inactiveTabsAreEnabled = containerView.context.settings().inactiveTabsAreEnabled
 
         val selectedTab = browserStore.state.selectedNormalTab ?: return
         // It's safe to read the state directly (i.e. won't cause bugs because of the store actions
```

Comment:
This comment seems to be out of place.  Do we know where this comment should be attached to?

Reply:
It probably belonged to `focusGroupTabId`, so I'll remove it since that's gone now. Good catch